### DISCUSSION TITLE
看到路边有个遥控器，无意按了一下,青衣就成这样了，大家有什么头绪吗

### DISCUSSION THREAD

**Discussion by Crafty-rabbit** [share121/inter-knot#1331] [其他]
看到路边有个遥控器，按了一下青衣就成这样了，大家有什么头绪吗
![dc046790acdd445219d15f58e54e7c2](https://github.com/user-attachments/assets/03f811ef-a8c3-4000-930e-97675a8dfed9)

**Comment by Qiling-XwX** [comment]
是起爆器罢！

**Comment by lllliufeng** [comment]
是EMP罢！快去请治安局！

**Comment by GanlinSama** [comment]
哎呀，看来是被EMP影响到了，可能内部结构有哪里坏掉了，送来我这儿让我看看怎么修

**Comment by wugan-bot156** [comment]
💻🦐💺

**Comment by ZhuYuan-148** [comment]
袭警！这是袭警！

**Comment by zzziloveyou** [comment]
现在跟青衣解释说不定还能来得及。。。上一个已经牢底坐穿了，你还好吗？

**Comment by dengyunzhe** [comment]
乱按emp可是要被青衣大人狠狠惩罚的

**Comment by warmgoose** [comment]
虽然很心疼，但是很好奇青衣姐姐维持运作是靠充电还是进食呢。。

**Comment by 0721Lover** [comment]
芝士成人的遥控器吧
